src/plugins/legacy/polygonRoi/polygonEventFilter.cpp

```diff
@@ -31,7 +31,7 @@
 #include <itkComposeImageFilter.h>
 
 polygonEventFilter::polygonEventFilter(medAbstractImageView *view) :
-        medViewEventFilter(), currentView(view), cursorState(CURSORSTATE::MOUSE_EVENT), isRepulsorActivated(false)
+        medViewEventFilter(), currentView(view), cursorState(CURSORSTATE::CS_MOUSE_EVENT), isRepulsorActivated(false)
```

Comment:
MOUSE_EVENT already exist on Windows environment.
Then I prefix it by 'CS_' for  cursorState to can compile.
Change the name to a better if you want.